Src/Plugins/Security/SEC101_026.MailgunApiCredentialsValidator.cs

```diff
@@ -15,13 +15,25 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
 {
     public class MailgunApiCredentialsValidator : DynamicValidatorBase
     {
-        protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary<string, FlexMatch> groups)
+        internal static HttpRequestMessage GenerateRequestMessage(string id, string secret, string scanIdentityGuid)
         {
-            if (!groups.TryGetNonEmptyValue("id", out FlexMatch id) ||
-                !groups.TryGetNonEmptyValue("secret", out FlexMatch secret))
+            string credentials = $"api:{secret}";
+            byte[] bytes = Encoding.UTF8.GetBytes(credentials);
+            credentials = Convert.ToBase64String(bytes);
+
+            using var request = new HttpRequestMessage(HttpMethod.Post, $"https://api.mailgun.net/v3/{id}.mailgun.org/messages");
```

Comment:
>using [](http://example.com/codeflow?start=12&length=5)

dont add the using here.
if u do that, when we run this, it will throw an exception.